__jest__/jest.transform.js

```diff
@@ -1,6 +1,9 @@
 // Custom Jest transform implementation that wraps babel-jest and injects our
 // babel presets, so we don't have to use .babelrc.
 
+/*eslint no-undef: "error"*/
+/*eslint-env node*/
+
 module.exports = require('babel-jest').createTransformer({
-  presets: ["env", "react", "stage-3"],
+  presets: ['@babel/preset-env', '@babel/preset-react']
```

Comment:
this is not exactly what I meant. in the other webpack we now doing:
```js
plugins: [
                    '@babel/plugin-transform-runtime',
                    '@babel/plugin-proposal-class-properties'
                ]
```
so I think it needs to be:
```js
presets: ['@babel/preset-env', 
                '@babel/preset-react', 
                plugins: [
                    '@babel/plugin-transform-runtime',
                    '@babel/plugin-proposal-class-properties'
                ]]
```
but you should test it.

Reply:
Fixed, thanks!